### PR TITLE
Generated v0.2.0 release

### PR DESCRIPTION
**Pull request by ellis-bigelow**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kfserving/408)

### CODE REVIEW

**Comment by yuzisun** [comment]
The manifests looks good, should we also cut a release branch?

**Comment by yuzisun** [comment]
note that @rakelkar 's PR https://github.com/kubeflow/kfserving/pull/339 has CRD changes, should we wait?

**Comment by rakelkar** [comment]
Also related: 
* Update docs to reflect new URL structure: https://github.com/kubeflow/kfserving/issues/413
* TRTIS not supported (need to update docs): https://github.com/kubeflow/kfserving/issues/271
* ONNX image needs to be updated to version that supports the v1 protocol: https://github.com/kubeflow/kfserving/issues/415

**Comment by yuzisun** [comment]
@ellis-bigelow Can you help regenerate the deployment yaml to reflect the latest changes?

**Comment by johnugeorge** [comment]
If all major manifest changes are in, can we merge this PR ?   We should make sure that kubeflow/manifests one matches the one in the repo.

**Comment by ellis-bigelow** [comment]
Will rgenerate

File: install/0.2.0/kfserving.yaml [hunk 1]
```diff
@@ -0,0 +1,948 @@
...
+            "image": "gcr.io/kfserving/xgbserver",
+            "defaultImageVersion": "latest",
+            "allowedImageVersions": [
+               "latest",
+               "0.2.0"
+            ]
+        },
+        "pytorch": {
+            "image": "gcr.io/kfserving/pytorchserver",
+            "defaultImageVersion": "latest",
```
Comment (yuzisun) [inline install/0.2.0/kfserving.yaml]: let's default to 0.2.0

File: install/0.2.0/kfserving.yaml [hunk 2]
```diff
@@ -0,0 +1,948 @@
...
+            "image": "gcr.io/kfserving/sklearnserver",
+            "defaultImageVersion": "latest",
+            "allowedImageVersions": [
+               "latest",
+               "0.2.0"
+            ]
+        },
+        "xgboost": {
+            "image": "gcr.io/kfserving/xgbserver",
+            "defaultImageVersion": "latest",
```
Comment (yuzisun) [inline install/0.2.0/kfserving.yaml]: let's default to 0.2.0

File: install/0.2.0/kfserving.yaml [hunk 3]
```diff
@@ -0,0 +1,948 @@
...
+  ingress: |-
+    {
+        "ingressGateway" : "knative-ingress-gateway.knative-serving",
+        "ingressService" : "istio-ingressgateway.istio-system.svc.cluster.local"
+    }
+  predictors: |-
+    {
+        "tensorflow": {
+            "image": "tensorflow/serving",
+            "defaultImageVersion": "latest",
```
Comment (yuzisun) [inline install/0.2.0/kfserving.yaml]: default to 1.14.0 and 1.14.0-gpu ?

File: install/0.2.0/kfserving.yaml [hunk 4]
```diff
@@ -0,0 +1,948 @@
...
+               "1.12.0-gpu",
+               "1.13.0",
+               "1.13.0-gpu",
+               "1.14.0",
+               "1.14.0-gpu"
+            ]
+        },
+        "onnx": {
+            "image": "mcr.microsoft.com/onnxruntime/server",
+            "defaultImageVersion": "latest",
```
Comment (yuzisun) [inline install/0.2.0/kfserving.yaml]: default to v0.5.0

File: install/0.2.0/kfserving.yaml [hunk 5]
```diff
@@ -0,0 +1,948 @@
...
+            "allowedImageVersions": [
+               "latest",
+               "v0.5.0"
+            ]
+        },
+        "sklearn": {
+            "image": "gcr.io/kfserving/sklearnserver",
+            "defaultImageVersion": "latest",
```
Comment (yuzisun) [inline install/0.2.0/kfserving.yaml]: default to 0.2.0

**Comment by yuzisun** [comment]
@ellis-bigelow let's default to the pinned image versions every time making the release, in config it is set to latest for development purpose.

File: install/0.2.0/kfserving.yaml (hunk 3 above)
Comment (ellis-bigelow) [reply install/0.2.0/kfserving.yaml]: Hmm -- not sure I want to couple this.

**Comment by ellis-bigelow** [comment]
> @ellis-bigelow let's default to the pinned image versions every time making the release, in config it is set to latest for development purpose.

Mind elaborating?

**Comment by yuzisun** [comment]
> > @ellis-bigelow let's default to the pinned image versions every time making the release, in config it is set to latest for development purpose.
> 
> Mind elaborating on why we want this? It's a bit challenging to script ;)

We just need to default to a safe version instead of latest since our latest builds everytime master updates and for tensorflow it is a pointer to the latest version, we do not want either case on prod environment since image can suddenly change when latest updates. Can we match up the default versions we set in kubeflow manifests (https://github.com/kubeflow/manifests/pull/461/files#diff-e2869cfdbfc11d64444ef14b8bd35efdR10), WDYT?

File: install/0.1.0/kfserving.yaml [hunk 6]
```diff
@@ -0,0 +1,942 @@
...
+    istio-injection: disabled
+  name: kfserving-system
+---
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  creationTimestamp: null
+  labels:
+    controller-tools.k8s.io: "1.0"
+  name: inferenceservices.serving.kubeflow.org
```
[the remark this answers is not in the document]
Comment (ellis-bigelow) [reply install/0.1.0/kfserving.yaml]: eeeeeeesh hahaha

**Comment by yuzisun** [comment]
here we go!

/lgtm
/approve

**Pull request merged**